src/test/scala/mesosphere/marathon/core/task/tracker/impl/InstanceCreationHandlerAndUpdaterDelegateTest.scala

```diff
@@ -5,14 +5,13 @@ import akka.Done
 import akka.actor.Status
 import akka.testkit.TestProbe
 import mesosphere.AkkaUnitTest
-import mesosphere.marathon.test.SettableClock
 import mesosphere.marathon.core.instance.TestInstanceBuilder
 import mesosphere.marathon.core.instance.TestInstanceBuilder._
-import mesosphere.marathon.core.instance.update.{ InstanceUpdateEffect, InstanceUpdateOperation }
+import mesosphere.marathon.core.instance.update.{InstanceUpdateEffect, InstanceUpdateOperation}
 import mesosphere.marathon.core.task.Task
 import mesosphere.marathon.state.PathId
-import mesosphere.marathon.test.MarathonTestHelper
-import org.apache.mesos.Protos.{ TaskID, TaskStatus }
+import mesosphere.marathon.test.{MarathonTestHelper, SettableClock}
```

Comment:
nit: there should be spaces I guess.

Reply:
Yeah, we should probably run the formatter. I recall there's a few places where code was merged to 1.5 without the formatter run.